.github/workflows/issue.yml

```
@@ -1,13 +1,66 @@
+
+  Issue-Greeting:
+    name: Greeting Message to User
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          ignore-comments: true
-          default-labels: '["unapproved"]'
+          issue-message: "Congratulations on making your first Issue! :confetti_ball: If you haven't already, check out our [Contributing Guidelines](https://github.com/PalisadoesFoundation/talawa-admin/blob/develop/CONTRIBUTING.md) and [Issue Reporting Guidelines](https://github.com/PalisadoesFoundation/talawa-admin/blob/develop/ISSUE_GUIDELINES.md) to ensure that you are following our guidelines for contributing and making issues."
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)